docs/src/concepts.md

```diff
@@ -0,0 +1,156 @@
+experimental interest. These segmented chunks can then be stored across all occurences
+of a particular event and then can be averaged across all related trials containing the
+event type. Typical common events of interest include stimlus or trial onset and motor
+response onsets(button press, eye movements, lever licks). ERPs can ultimately be used
+to identify patterns of neural activity associated with the responses to the events of
+interest.
+
+## Terminology
+
+The language below is tailored to the dependent variable in many neuroscience
+experiments, behavior. These concepts could be restated in reference to any modality.
+
+```text
+|----------------------------------------------------------------------------|
+|-------------------------------- Session ---------------------------------|__
+|------------------------------- Recording ------------------------------|____
+|----- Block 1 -----|______|----- Block 2 -----|______|----- Block 3 -----|___
+| Trial 1 || Trial 2 |____| Trial 3 || Trial 4 |____| Trial 5 |____| Trial 6 |
+|_|e1|_|e2||e3|_|e4|__|e5|__|e6||e7||e8||e9||e10||e11|____|e12||e13|_________|
+|----------------------------------------------------------------------------|
+```
+
+- A **Session** is period during which data is collected.
+- A **Recording** is some source of data tied to a single modality (e.g., behavior).
+This may or may not fully capture the session depending on recording latencies or
+equipment malfunctions.
+- **Block** and **Trial** are non-instantaneous subsets of Session whose traits often
+repeat across instances. These periods may be combined or contrasted in downstream
+analyses.
+- Trials may occur within or extend to the intervals between **Blocks**.
+- An **Event** (represented with e above) is an optionally instantaneous occurrence
+during a **Session**.
+- Projects may differ in their need to record event duration (e.g., onset versus
+duration of subject behavior)
+- **Events** may occur during other categories, or during continuously recorded behavior.
+
+Features of Element Event include:
+
+- Pairing of upstream sessions with behavioral recordings
+- Multiple recorded attributes for phases of interest (see Attribute part tables for
+Block and Trial)
+- Defining Trial and Event Types as lookup tables
+- Optionally activating only the event schema for event-based recording, without Trial
+and Block phases.
+- An AlignmentEvent table to define the window of interest relative to specific event
+types, such as Peristimulus Time Histogram (PSTH) analysis commonly used in
+electrophysiology studies.
+
+## Key Partnerships
+
+DataJoint has partnered with the following teams to interview key members, and develop
+individualized pipelines. By comparing across use-cases, the DataJoint team has
+developed a highly adaptable workflow to meet most needs, and trialize analyses within
+an existing DataJoint workflow.
+
+- International Brain Lab
```

Comment:
Do the dashes render well as bullet points? I had to use `+`

Reply:
Yup - `-`, `+`, and others (I think `*`) are equivalent. Some markdown IDE extensions will give automatically pick one or another for intent tiers, but I don't think we have a system.